RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -11,6 +11,38 @@
 _pfZeroSuppressionThresholds_EEminus_2017 = pfZeroSuppressionThresholds_EEminus
 _pfZeroSuppressionThresholds_EEplus_2017 = _pfZeroSuppressionThresholds_EEminus_2017
 
+# A
```

Comment:
please add some comments to describe the purpose of these 3 alternative parameters.
IIUC, none of these will make it to the standard reco until the studies are complete.

Reply:
I see that comments were added to describe A,B,C in HLTrigger/Configuration/python/customizeHLTforCMSSW.py.
For completeness, please add something similar here as well

Reply:
Sure.
For sake of completeness and future references, I add here a link to a presentation with the details:
# For additional information: https://indico.cern.ch/event/704940/contributions/2892425/attachments/1602745/2541631/2018EcalChanges_jetMet.pdf